rescript

```diff
@@ -67,66 +67,64 @@ process.on("SIGUSR2", exitProcess);
 process.on("SIGTERM", exitProcess);
 process.on("SIGHUP", exitProcess);
 
-const process_argv = process.argv;
-const maybeSubcommand = process_argv[2];
+const args = process.argv.slice(2);
+const helpArgIndex = args.findIndex(arg => /help|-h|-help|--help/.test(arg));
```

Comment:
I'd avoid using regex for the check

```suggestion
const helpArgIndex = args.findIndex(arg => ["help", "-h", "-help", "--help"].includes(arg));
```

Or even 4 strict equal checks

Reply:
Regex is difficult and slow.

About being difficult: Your regex needs to check that the result is within word boundaries `/\b(help|-h|-help|--help)\b/`, otherwise it'll match `helper` as `help`.

About being slow is not critical when called once, but would be nice to stick to faster alternative:

<img width="1142" alt="image" src="https://github.com/rescript-lang/rescript-compiler/assets/49292466/f5c28d76-1bdd-44bb-854b-e5bee6999708">